### DISCUSSION TITLE
关于部署的小问题

### DISCUSSION THREAD

**Discussion by Teylusu** [simple-robot/Discussions#6] [Q&A] [answered]
本地运行一切正常，我将程序部署至服务器后监听功能全部失效，不知是什么问题。

**Comment by ForteScarlet** [comment, marked as answer]
大概率与 [这个问题](https://www.yuque.com/simpler-robot/simpler-robot-doc/ie2t71) 相似。
可以测试一下 **私聊事件** 是否有效，或者通过定时任务（或其他非监听函数内）来尝试获取bot的信息（例如获取好友列表等）。

也可以换几个账号做尝试。

**Reply by Teylusu** [reply]
昨晚睡前挂在服务器上了，早上起来发现定时任务的群消息发出来了两条。
然后就试了试私聊，发现私聊的监听还是不行，之后定时任务也刮掉了。
应该还是TX的风控把。
